src/uwtools/resources/jsonschema/chgres-cube.jsonschema

```diff
@@ -0,0 +1,205 @@
+                    },
+                    "lai_from_climo": {
+                      "type": "boolean"
+                    },
+                    "minmax_vgfrc_from_climo": {
+                      "type": "boolean"
+                    },
+                    "mosaic_file_input_grid": {
+                      "type": "string"
+                    },
+                    "mosaic_file_target_grid": {
+                      "type": "string"
+                    },
+                    "nsoill_out": {
+                      "type": "integer"
+                    },
+                    "nst_files_input_grid": {
+                      "type": "string"
+                    },
+                    "orog_dir_input_grid": {
+                      "type": "string"
+                    },
+                    "orog_dir_target_grid": {
+                      "type": "string"
+                    },
+                    "orog_files_input_grid": {
+                      "type": "string"
+                    },
+                    "orog_files_target_grid": {
+                      "type": "string"
+                    },
+                    "regional": {
+                      "type": "integer"
+                    },
+                    "sfc_files_input_grid": {
+                      "type": "string"
+                    },
+                    "sotyp_from_climo": {
+                      "type": "boolean"
+                    },
+                    "tg3_from_soil": {
+                      "type": "boolean"
+                    },
+                    "thomp_mp_climo_file": {
+                      "type": "string"
+                    },
+                    "tracers": {
+                      "type": "array"
+                    },
+                    "tracers_input": {
+                      "type": "array"
+                    },
+                    "varmap_file": {
+                      "type": "string"
+                    },
+                    "vcoord_file_target_grid": {
+                      "type": "string"
+                    },
+                    "vgfrc_from_climo": {
+                      "type": "boolean"
+                    },
+                    "vgtyp_from_climo": {
+                      "type": "boolean"
+                    },
+                    "wam_cold_start": {
+                      "type": "boolean"
+                    },
+                    "wam_parm_file": {
+                      "type": "string"
+                    }
+                  },
+                  "type": "object"
+                }
+              },
+              "required": [
+                "config"
+              ],
+              "type": "object"
+            }
+          },
+          "type": "object"
+        },
+        "run_dir": {
+          "type": "string"
+        }
+      },
+      "required": [
+        "execution",
+        "run_dir"
+      ],
+      "type": "object"
+    }
+  },
+  "type": "object"
+}
```

Comment:
Maybe it's still WIP, or maybe happening in @WeirAE 's branch/PR, but it would be valuable to have some granular test for this schema in `uwtools.tests.test_schemas`. Without those it's IMO hard to tell if the schema really does what's intended, as it's hard to test so many possible combinations via unit tests of other code. It might not need much, but I see now that the one I wrote for  `sfc_climo_gen` could probably have somewhat better coverage: There's probably no sense in testing that something like
```
"input_vegetation_type_file": {
  "type": "string"                                                                                                                                                                                                   
}
```
doesn't accept, say, an `int` value, but possibly something like
```
maximum_snow_albedo_method": {
  "enum": [                                                                                                                                                                                                          
    "bilinear",                                                                                                                                                                                                      
    "conservative"                                                                                                                                                                                                   
  ]                                                                                                                                                                                                                  
}
```
could use tests against misspellings. It's hard to know where to draw the line, but today I feel like I could have added a few more tests of `sfc_climo_gen.jsonschema`. `chgres-cube.jsonschema` probably deserves at least a block in `test_schemas.py` to make it obvious that we didn't just overlook it.